shim.js

```diff
@@ -32,6 +32,20 @@ if (
 
 global.storage = storage;
 
+Object.defineProperty(global, 'android', {
+  get: () => ReactNative.Platform.OS === 'android',
+  set: () => {
+    throw new Error('Trying to override internal Rainbow var');
+  },
+});
+
+Object.defineProperty(global, 'ios', {
+  get: () => ReactNative.Platform.OS === 'ios',
+  set: () => {
+    throw new Error('Trying to override internal Rainbow var');
+  },
+});
+
```

Comment:
Should we add `Object.freeze()` so it's not possible to be changed on runtime (maybe a dep by using the same name for a global)

Reply:
not really. If the lib will try to use this name it will throw the exception! Freezing `global` is not the best idea I think.